test/quantization/fx/test_quantize_fx.py

```diff
@@ -1778,7 +1778,7 @@ def forward(self, x):
                  None),
             ]
             for (ModuleClass, module_constructor_inputs,
-                 inputs, quantized_node, weight_prepack_node) in tests:
+                 inputs, _quantized_node, weight_prepack_node) in tests:
```

Comment:
_ ?

Reply:
To explain, the linter is OK with unaccessed variables in for loops if they start with an underscore. In a few cases it seemed worth it to keep the variable name, if the variable was something non-obvious, to make the code more readable. But I can change these to just _ if that would be better.